json_serializable/test/integration/json_keys_example.dart

```diff
@@ -23,6 +23,6 @@ class Model {
   String get fullName => '$firstName $lastName';
 
   Map<String, Object?> toJson() => _$ModelToJson(this);
-
-  static const jsonKeys = ModelJsonKeys;
 }
+
+typedef ModelJsonKeys = _$ModelJsonKeys;
```

Comment:
Smart, i like the idea

Reply:
@ChreSyr  would be really nice to have this.